tower.yml

```diff
@@ -0,0 +1,5 @@
+reports:
+  multiqc_report.html:
+    display: "MultiQC HTML report"
+  samplesheet.csv:
+    display: "Auto-created samplesheet with collated metadata and FASTQ paths"
```

Comment:
Can you details the simple files created by the pipeline there?

Reply:
resolved (i believe) in [b326514](https://github.com/nf-core/callingcards/pull/14/commits/b326514e05955ec7eb40fba59a4356d65bbd45e7)